test/e2e/autoscaling/custom_metrics_stackdriver_autoscaling.go

```diff
@@ -217,6 +227,26 @@ var _ = SIGDescribe("[HPA] Horizontal pod autoscaling (scale resource: Custom Me
 			hpa:             externalHPA(f.Namespace.ObjectMeta.Name, metricTargets)}
 		tc.Run()
 	})
+
+	ginkgo.It("should scale down to 0 with Custom Metric of type Object from Stackdriver [Feature:CustomMetricsAutoscaling]", func() {
+		fg.Set("HPAScaleToZero=true")
+		e2eskipper.SkipUnlessFeatureGateEnabled("HPAScaleToZero")
```

Comment:
what are these two lines doing? it looks like this sets a feature gate to true, then asks to skip the test unless the feature gate is set to true? that doesn't seem very useful... am I misreading that?